spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelErrorConfigurer.java

```diff
@@ -0,0 +1,163 @@
+package org.springframework.cloud.stream.binder.rabbit;
```

Comment:
Copyright is missed. Again the Checkstyle responsibility.
I don't tell to do that right now, but our code is getting dirtier and dirtier...